.travis.yml

```diff
@@ -2,24 +2,31 @@ language: cpp
 os: linux
 compiler: gcc
 
+cache:
+- directories:
+  - $HOME/.cache
```

Comment:
I wanted to show the best practice of using the cache. No real dependencies are used here. Shall I add to the install script that it usually is rather a combination of `wget, configure --prefix $HOME/.cache, make install`?

Reply:
> Like 13d5ecd ?

I like referencing the documentation, but why not add it in the readme?

On second thought, the whole example comment seems overkill.
Just mentioning it is representing external dependencies build from source / on the CI, should suffice.

Since there is no fits-all implementation, it seems better not to get too specific. Thus making the 'fake' implementation a good placeholder.

Reply:
fb67edd ?

Reply:
Good point: https://github.com/codecov/example-cpp11-cmake/pull/13/commits/fb67edd4842426e191d70a3742dd26e30a50ae31

Reply:
Uhm, I think you forgot to push the commit.

Reply:
Oh, sorry. Done now